ocamltest/builtin_actions.ml

```diff
@@ -305,6 +305,13 @@ let has_symlink = make
     "symlinks available"
     "symlinks not available")
 
+let not_root = make
+  ~name:"not_root"
```

Comment:
I vote for `not-root`, the form `not-foo` is common and `not_foo` is inconsistent.

Reply:
done